src/pages/index.astro

```diff
@@ -1,8 +1,28 @@
 ---
 import MainHero from '../components/MainHero.astro';
 import Layout from "../layouts/Layout.astro";
+import WorkChart from "../components/WorkChart.astro";
 ---
 
 <Layout title="STRENSIQ | HCP">
+
   <MainHero />
+
+  <section class="lets-work wrapper">
+    <h1 class="lets-work__title">Let's get to work</h1>
+
+    <div class="lets-work__charts">
+      <WorkChart img="/assets/images/glass.png" href="#"> 
+        <p>Learn more about hypophosphatasia</p>
+    </WorkChart>
+
+    <WorkChart img="/assets/images/tool.png" href="#"> 
```

Comment:
PLease make sure to make properties optionals if you are not always gonna need it to render the component.